dashboard/pkg/epinio/store/epinio-store/actions.ts

```diff
@@ -3,12 +3,15 @@ import { handleSpoofedRequest } from '@shell/plugins/dashboard-store/actions';
 import { classify } from '@shell/plugins/dashboard-store/classify';
 import { normalizeType } from '@shell/plugins/dashboard-store/normalize';
 import { NAMESPACE_FILTERS } from '@shell/store/prefs';
-import { base64Encode } from '@shell/utils/crypto';
 import { createNamespaceFilterKeyWithId } from '@shell/utils/namespace-filter';
 import { parse as parseUrl, stringify as unParseUrl } from '@shell/utils/url';
+import epinioAuth, { EpinioAuthTypes } from '../../utils/auth';
+
 import {
   EpinioInfo, EpinioVersion, EPINIO_MGMT_STORE, EPINIO_PRODUCT_NAME, EPINIO_STANDALONE_CLUSTER_NAME, EPINIO_TYPES
 } from '../../types';
+import EpinioCluster from '../../models/cluster';
+import { RedirectToError } from '@shell/utils/error';
```

Comment:
`RedirectToError` is not exported by `@shell/utils/error`. Do we need to update the `shell` dependency maybe ?

This is causing an error during logout:

```
Loading error Error: _shell_utils_error__WEBPACK_IMPORTED_MODULE_30__.RedirectToError is not a constructor
    at Store.setError (index.js:654:1)
    at wrappedMutationHandler (vuex.esm.js:844:1)
    at commitIterator (vuex.esm.js:466:1)
    at Array.forEach (<anonymous>)
    at vuex.esm.js:465:1
    at Store._withCommit (vuex.esm.js:624:1)
    at Store.commit (vuex.esm.js:464:1)
    at Store.boundCommit [as commit] (vuex.esm.js:409:1)
    at _callee$ (authenticated.js:509:1)
    at tryCatch (regeneratorRuntime.js:44:1)
```

Reply:
That was https://github.com/rancher/dashboard/pull/9941, which was blocked on branching. If you `yarn link` to `master` the error should resolve itself. 

Now that we've forked it has merged, so we can do another shell release

Reply:
i've now published a new shell and referenced that from the extension